catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -113,9 +115,10 @@ const SearchForm = ({
                   action: 'clear search',
                   label: 'works-search'
                 });
-                const link = worksUrl({query: null, page: null});
+
                 setQuery('');
-                Router.push(link.href, link.as);
+
+                searchInput.current.focus();
```

Comment:
As `searchInput` has the potential to be null (the initial value) - I think you need to run a null check here.

Reply:
correct! Thanks